lib/routes/email.js

```diff
@@ -33,7 +33,15 @@ module.exports = {
         return reply(new AppError.authError('network error'));
       }
       if (res.statusCode >= 400) {
-        logger.error('request.auth_server.fail', { code: res.statusCode });
+        body = body && body.code ? body : { code: res.statusCode };
+        // It's possible for the account to be deleted on the auth-server
+        // but its oauth tokens are still valid, pending cleanup.
+        // We just treat it as if auth failed.
+        if (body.errno === 102) {
+          logger.info('request.auth_server.fail', body);
+          return reply(new AppError.unauthorized(body.message));
+        }
+        logger.error('request.auth_server.fail', body);
         return reply(new AppError.authError('auth server error'));
```

Comment:
The other possible 4XXs from that endpoint are:

* a `401` if the bearer token is invalid; this should be passed directly back to the client
* various `400` errors due to malformed requests

Seems we're better off turning them into 400s rather than 503s, I'll update the PR.